packit_service/service/api/runs.py

```diff
@@ -126,6 +128,24 @@ def process_runs(runs):
                 response_dict,
             )
 
+        # handle VM image builds
+        for vm_image_build_id in set(
+            flatten_and_remove_none(pipeline.vm_image_build_id)
+        ):
+            vm_image_build = VMImageBuildTargetModel.get_by_id(vm_image_build_id)
+            response_dict["vm_image_build"].append(
+                {
+                    "packit_id": vm_image_build_id,
+                    "target": vm_image_build.target,
+                    "status": vm_image_build.status,
+                }
+            )
+            if "trigger" not in response_dict:
+                response_dict["time_submitted"] = optional_timestamp(
+                    vm_image_build.build_submitted_time
+                )
+                response_dict["trigger"] = get_project_info_from_build(vm_image_build)
```

Comment:
could we also have this in a separate method?

Reply:
Ok, let me move it.